test/Sentry.Tests/ScopeTests.cs

```diff
@@ -232,6 +233,51 @@ public void AddAttachment_AddAttachments()
         scope.Attachments.Should().Contain(attachment2, "Attachment2 was not found.");
     }
 
+    [Fact]
+    public void Clear_SetsPropertiesToDefaultValues()
+    {
+        // Arrange
```

Comment:
Could use something like Bogus/AutoBogus here to generate fakes.

Reply:
Neat idea! But let's think about that as a separate future endeavor. 😃